apps/updownload/src/components/Import.vue

```diff
@@ -7,7 +7,7 @@
       <div v-if="taskUrl">
         <h4>Progress of current upload:</h4>
         <ul class="fa-ul">
-          <Task :task="task" />
+          <SubTask :task="task" />
```

Comment:
Should be still 'Task' right?

Reply:
There was a local component in this repo that was called task but did the same thing as the "SubTaks" component in the component-library. I opted for using the component from the library and removing the local one.

Reply:
Now also removed the unused component. ;)